### PR TITLE
YouTube video page

### PR DESCRIPTION
**Pull request by elasfarc**
# Youtube video home page

In this project, we worked on the following:

- Embed a YouTube video on the page
- Add channel information, video description, and the comments section
- Create a navigation bar with a logo, a search bar, and the user avatar
- Create a sidebar for recommended videos
- Add a footer 
- Style it as an original YouTube page
- Add a README

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## What a remarkable project you have built 💯! This is impressive 🤩👌🧨!

### But in order to get it approved, we need to work on some changes. Let's see...

- Let's make the README more professional. Right now it's looking bland. Remember, READMEs are very important part of a project. It provides valuable information about the project. You can use this README template: https://github.com/microverseinc/readme-template. Fill up the sections that are necessary for your project and remove the other sections. For eg., `Install`, `Usage`, `Run tests` and `Deployment` is irrelevant to your project, so you can remove them.

- Remember to `.gitignore` the files that are irrelevant to your project. Please add a `.gitignore` file and put the `.DS Store` file in that. Also, this file is present in the assets folder. Be sure to remove all of them.
image

- There are lots of errors and warnings in your HTML file. Please go over to https://validator.w3.org/#validate_by_input and paste your HTML code. Go through the errors and warnings and fix them.

image

Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+  background-color: rgb(250, 250, 250);
+  color: rgb(115, 115, 115);
+  font-size: 11px;
+  padding: 4px 7px;
+}
+
+.subscribe{
+  /*display: flex;
+  justify-content: space-evenly;*/
+}
```
Comment (bertil291utn) [inline homepage.css]: Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history (screenshot from Pathwright)
❌ **commented piece of codes** `/* .nav { color: red;} */`
✔ **comments about code** `/*Navigation */`
Comments about the code is okay and should only be tagged as OPTIONAL if it's unnecessary and please provide details why it is unnecessary.

File: homepage.html [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+          <div class="preview-info">
+              <a href="#"><p class="preview-videoTitle"><strong>Contributing to Open Source Part II: The Real Way</strong></p></a>
+              <a href="#"><p class="preview-channel-name">The Odin Project</p></a>
+            <p class="preview-views">29,985 views</p>
+          </div>
+        </div>
+
+
+        <ul>
+          <li><hr id="previewContainer-hLine"/></li>
```
Comment (bertil291utn) [inline homepage.html:169]: ## Using an ID Selector for Multiple Elements[1]
When writing CSS, there should only be one ID in the HTML. Always use the class attribute for multiple elements.
- [ ] Use `class` instead `id`

_[1] [Good and Bad CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: homepage.html [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+       <title>Contributing to Open Source Part I: The Easy Way
</title>
+       <script src="https://kit.fontawesome.com/e53eea7683.js" crossorigin="anonymous"></script>
+       <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+       <link rel="stylesheet" href="./homepage.css">
+   </head>
+
+   <body>
+     <nav>
+       <section class="menu-logo">
```
Comment (bertil291utn) [inline homepage.html]: ## Check usage of semantic HTML elements. 
Section tag need a h2-h6 tag to accomplish. Check html [validator](https://validator.w3.org/#validate-by-input)

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🔴  :shipit: ? Not yet

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

Your work is terrific you certainly put all your effort in this project, however, we need to make some changes.

- [x] Improve your README file. Try to follow this [template](https://github.com/microverseinc/readme-template)
- [ ] Your pull request comment should not be a copy-paste of your README file, please edit your pull request. Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

image

- [x] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [ ] Please work also with inline comments on this Pull request
<hr>


Guys work in the requested changes above and request code for another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Great job guys :+1: you have made a great youtube clone but you still need to fix one final think before your project gets approved :checkered_flag: .

- [ ] I still see that you still have the DS_Store file in your repository, please remove it I understand that you have added it to the .gitignore but you need to remove it first and push. then add the .gitignore file again.
image


### Please make all the required changes before you submit another code review.
## Happy Coding :tophat: 
_If you have any issue or need help don't hesitate to contact me on slack_

**Comment by bertil291utn** [comment]
**Status:** Approved ✅

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

 ## Ship it

image

Please merge to master branch and move on with the next assignment.

You're doing  a great job keep going 💻 🚀 🔥

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Pull request merged**